### DISCUSSION TITLE
Reducing the number of CI jobs per commit

### DISCUSSION THREAD

**Discussion by pjb7687** [gnudatalanguage/gdl#925] [Ideas]
Hi all,

Currently, the CI script submits 20 jobs per commit, which I think way too much. Of course not very good for the environment. So I consider reducing the number of jobs per commit, e.g. only running 64-bit full Debug builds.

To complement, I think it would be better to introduce nightly or weekly builds, running all release builds every night or every week. What do you think?

**Comment by pjb7687** [comment]
Also, I think it is not worth supporting 32-bit Windows builds today... IMHO in 2021, nobody uses 32-bit Windows.

**Comment by GillesDuvert** [comment]
completely agree, one full build per system (64 bits) would even be enough for everyday life, then a weekly build would probably suffice for all other purposes. codecov should be used only weekly too.
